mos-platform/osi-c1p/crt0.c

```diff
@@ -0,0 +1,10 @@
+extern void __clrscr(void);
+
+asm(
+  ".section .init.40,\"axR\",@progbits\n"
+  "  jsr __late_init\n"
```

Comment:
You might consider just `jsr __clrscr`, at least until there's something else in `__late_init`. I took a look at the NES target; we could probably do that there too (but no action required for this PR); if I recall correctly, there used to be more stuff in the NES `__late_init`.

In that case, you could just make this a .s file, which seems a bit cleaner than inline assembly.

Reply:
I replaced the crt0.c module with a crt0.s module that calls ``__clrscr`` directly.